README.md

```diff
@@ -101,6 +101,33 @@ Example Response body:
 {“journeyStartUrl” : "/testUrl"}
 ```
 
+### POST /limited-liability-partnership-journey
+
+---
+Creates a new journey for a Limited Liability Partnership, storing the journeyConfig against the journeyId.
+#### Request:
+optServiceName will default to `Entity Validation Service` if the field is not provided.
+
+All other fields must be provided.
+
+```
+{
+   "continueUrl" : "/test",
+   "optServiceName" : "Service Name",
+   "deskProServiceId" : "abc",
+   "signOutUrl" : "/sign-out",
+}
+```
+
+#### Response:
+Status: **Created(201)**
+
+Example Response body:
+
+```
+{“journeyStartUrl” : "/testUrl"}
+```
```

Comment:
Guess we can keep this common for all endpoints

Reply:
I see your point and have updated the readme. However, if we were using a more formal API definition i suspect we would need to define the response for each endpoint.

Reply:
I agree in that case would have made sense